src/components/FuncCard.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import "./FuncCard.css";
+import FuncCode from './FuncCode';
+import FuncParams from './FuncParams';
+import {
+  Card,
+  CardActions,
+  CardText,
+  FlatButton,
+  CardTitle,
+} from 'material-ui';
+
+class FuncCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      params: [],
+      result: '',
+      showCode: false,
+      code: '',
+    };
+
+    this.fetchCode(props.funcName);
```

Comment:
This is a good opp to advantage of lifecycle components.   This should go somewhere in the code where props gets updated.  I don't think it would be `componentDidMount` because props probably wouldn't be ready yet.  Maybe `componentDidUpdate` or `componentWillReceiveProps`  You'll have to test out each one

Reply:
According to [docs](https://reactjs.org/docs/react-component.html#componentdidmount), `componentDidMount` is the good place for this:
>If you need to load data from a remote endpoint, this is a good place to instantiate the network request.